packages/state-transition/src/cache/epochTransitionCache.ts

```diff
@@ -143,12 +149,12 @@ export interface EpochTransitionCache {
    * | beforeProcessEpoch               | calculate during the validator loop|
    * | afterEpochTransitionCache                | read it                            |
    */
-  nextEpochShufflingActiveValidatorIndices: ValidatorIndex[];
+  nextShufflingActiveIndices: ValidatorIndex[];
```

Comment:
model this as `Uint32Array`, populated like below and to be shared between `EpochCache` and `EpochShuffling`